core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/utils/TableReference.java

```diff
@@ -14,55 +14,31 @@
  */
 package com.ericsson.bss.cassandra.ecchronos.core.utils;
 
+import java.util.UUID;
+
 /**
- * A class containing keyspace/table mapping to avoid passing around two strings to refer to one specific table.
+ * An interface containing keyspace/table mapping to avoid passing around two strings to refer to one specific table.
  */
-public final class TableReference
+public interface TableReference
 {
-    private final String myKeyspace;
-
-    private final String myTable;
-
-    public TableReference(String keyspace, String table)
-    {
-        myKeyspace = keyspace;
-        myTable = table;
-    }
-
-    public String getTable()
-    {
-        return myTable;
-    }
-
-    public String getKeyspace()
-    {
-        return myKeyspace;
-    }
-
-    @Override
-    public String toString()
-    {
-        return myKeyspace + "." + myTable;
-    }
-
-    @Override
-    public boolean equals(Object o)
-    {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-
-        TableReference that = (TableReference) o;
-
-        if (myKeyspace != null ? !myKeyspace.equals(that.myKeyspace) : that.myKeyspace != null) return false;
-        return myTable != null ? myTable.equals(that.myTable) : that.myTable == null;
-
-    }
-
-    @Override
-    public int hashCode()
-    {
-        int result = myKeyspace != null ? myKeyspace.hashCode() : 0;
-        result = 31 * result + (myTable != null ? myTable.hashCode() : 0);
-        return result;
-    }
+    /**
+     * Get the table id.
+     *
```

Comment:
Nit: Do we really need these descriptions? I suggest we either remove the description and leave the return or we remove it all together. I think most people can probably figure out what the function does just by the name of it.

Reply:
Done